blog/articles/event-drive-app-knative-eventing-kogito.md

```diff
@@ -0,0 +1,130 @@
+
+As part of the Serverless Workflow implementation, Kogito offers a Kubernetes Operator to deploy these workflows with Knative. The goal is to make it as simple as possible to deploy and manage user-defined workflows in  cloud environments. Knative Eventing plays a very important role in this scenario by providing the underlying infrastructure for event-driven architectures.
+
+### Kogito Serverless Workflow
+
+To demonstrate how the Kogito workflow implementation works on event-driven architectures, we will use the [Patient Onboarding example](https://github.com/kiegroup/kogito-examples/tree/master/serverless-workflow-functions-events-quarkus). In this example, we simulate a workflow used in hospitals to onboard new patients and assign them to the correct doctor.
+
+The following image taken from the [specification examples page](https://github.com/serverlessworkflow/specification/blob/master/examples/examples.md#New-Patient-Onboarding) illustrates this workflow:
+
+![Patient Onboarding workflow representation](https://raw.githubusercontent.com/serverlessworkflow/specification/master/media/examples/example-patientonboarding.png)*Patient Onboarding workflow representation*
+
+The workflow will start after receiving a [Cloud Event](https://github.com/cloudevents/spec) with patient information, and then will call three functions in a sequence which: (1) store the patient information, (2) assign to a doctor based on their symptoms, and (3) schedule an appointment with the assigned doctor.
+
+This is how the Patient Onboarding workflow looks like based on the specification:
+
+```yaml
+id: onboarding
+version: '1.0'
+name: Patient Onboarding Workflow
+states:
+- name: Onboard
+  type: event
+  start:
+    kind: default
+  onEvents:
+  - eventRefs:
+    - NewPatientEvent
+    actionMode: sequential
+    actions:
+    - functionRef:
+        refName: StoreNewPatient
+    - functionRef:
+        refName: AssignPatientToDoctor
+    - functionRef:
+        refName: SchedulePatientAppointment
+  end:
+    kind: default
+events:
+- name: NewPatientEvent
+  type: new.patient.events
+  source: "/hospital/entry"
+functions:
+- name: StoreNewPatient
+  operation: classpath:openapi.json#storeNewPatient
+- name: AssignPatientToDoctor
+  resource: classpath:openapi.json#assignPatientToDoctor
+- name: SchedulePatientAppointment
+  resource: classpath:openapi.json#schedulePatientAppointment
+```
+
+The above workflow definition is based on three parts. The first part is the “states” array which includes the workflow control-flow logic, in our case the “event” state which receives the new patient event and performs our three actions (service invocations).
+
+The second part is our event definitions (the “events” array), which are based on the CloudEvents specification, so there is a clear 1-1 mapping between how the events are defined in the workflows, and how they are represented with the CloudEvents format.
```

Comment:
```suggestion
The second part of the spec is the `events` array, which contains event definitions that are based on the CloudEvent specification. There is a 1:1 mapping between how the events are defined in the workflows, and how they are represented with the CloudEvent format.
```

Reply:
Same here, I think this paragraph is specific to the actual workflow definition shown, and not the specification itself. 
From the spec perspective valid workflows that don't consume events would not define the "events" array", so this is specifically describing the example workflow in the doc.